src/widgets/conditionalWidget/matrix1d.js

```diff
@@ -1,13 +1,16 @@
-const getPillarOptions = widgetData => (
-    widgetData.rows.map(r => ({
+const emptyObject = {};
+const emptyArray = [];
+
+const getPillarOptions = ({ rows = emptyArray } = {}) => (
+    rows.map(r => ({
         key: r.key,
         title: r.title,
     }))
 );
 
-const getSubpillarOptions = widgetData => (
-    widgetData.rows.reduce((acc, r) => [
-        ...r.cells.map(c => ({
+const getSubpillarOptions = ({ rows = emptyArray } = {}) => (
+    rows.reduce((acc, r) => [
+        ...((r || emptyObject).cells || emptyArray).map(c => ({
```

Comment:
r should have a default value instead
Avoid using variable names without meaning like 'r'